tools/nodered2mcu.js

```diff
@@ -470,7 +470,9 @@ export default class extends TOOL {
 				config.active = !!config.active;
 			} break;
 
-			case "function": {
+			case "function":
+			case "mcu_code":
+			{
```

Comment:
Since the "mcu_code" node is a duplicate of the "function", you can change the "mcu_code" to a "function" node here. That eliminates the need for "mcu_code" in Node-RED MCU.

```js
			case "mcu_code":
				config.type = "function";
				// fall through
			case "function": {
				const params = "node, context, flow, global, libs, env"
```

Reply:
At this point, it would be best if you could make a suggestion.
When the function and mcu_code nodes have different functions, it is better to separate them.
thank you.